ceph_installer/tests/controllers/test_agent.py

```diff
@@ -29,8 +29,9 @@ def test_index_post_is_denied(self, session, monkeypatch):
             expect_errors=True,
             extra_environ=dict(REMOTE_ADDR='192.168.1.1')
         )
-        assert result.status_int == 403
-        assert result.json['message'] == 'this resource does not allow non-local requests'
+        assert result.status_int == 200
```

Comment:
@ktdreyer the tests for the other endpoints verify that remote requests are given a ``403``, e.g. https://github.com/ceph/ceph-installer/blob/master/ceph_installer/tests/controllers/test_mon.py#L43

This only removed it for the ``/api/agent/`` endpoint.